client/src/components/Viewer/Viewer.js

```diff
@@ -169,7 +173,6 @@ function Viewer() {
           uploadStatus={isUploaded}
           email={shortEmail}
           uploadPercentage={uploadPercentage}
-          recentUploaded={recentUploaded}
```

Comment:
What was this added for previously? context needed

Reply:
As far i remember to get the last uploaded file but this method was discarded in between only.